docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -127,7 +127,8 @@ file via an expansion.
 Options:
 
 -   Checking **private** makes the variable redacted so the value won't
-    be visible on the projects page or by API routes.
+    be visible on the projects page or by API routes. Per  
+    [evergreen policy](https://mongodb.stackenterprise.co/questions/1232), private variables cannot be retrieved. 
```

Comment:
minor nit, we should capitalize Evergreen

Reply:
thanks, fixed